app/Interfaces/Interfaces.ts

```diff
@@ -1,10 +1,11 @@
-import type { ButtonMode, ButtonType } from '../Enums/Enums'
+import type { ButtonMode } from '../Enums/ButtonMode.enum';
+import { ButtonType } from '../Enums/ButtonType.enum';
```

Comment:
interface name should be ButtonPropsInterface and also directory should be app/Components/Button/interfaces/button-props.interface.ts